components/body-map.js

```diff
@@ -107,6 +105,9 @@ export function BodyMap({
 }) {
   const [highlightedOrgans, setHighlightedOrgans] = useState([]);
   function highlightOrgans(organ) {
+    if (!organ) {
+      setHighlightedOrgans([]);
+    }
     let organs = [organ];
```

Comment:
I think you need the rest of this code under else, so:
```javascript
if (!organ) {
  setHighlightedOrgans([]);
} else {
  let organs = [organ];
…
```
or it might be more natural to read:
```javascript
if (organ) {
  let organs = [organ];
…
} else {
  setHighlightedOrgans([]);
}
```

Reply:
agree!